Integrations/SlashNextPhishingIncidentResponse/SlashNextPhishingIncidentResponse.py

```diff
@@ -1156,6 +1151,60 @@ def download_text_command():
     })
 
 
+def api_quota():
+    """
+    Execute SlashNext's quota/status to get the quota status information
+    :return: Response of the SlashNext quota/status API
+    """
+    # Create the required data dictionary for Quota/Status
+    api_data = {}   # type: Dict[str, str]
+    response = http_request(endpoint=API_QUOTA, data=api_data)
+
+    if response.get('errorNo') != 0:
+        return_error('API Returned, {}:{}'.format(response.get('errorNo'), response.get('errorMsg')))
+
+    return response
+
+
+def api_quota_command():
+    """
+    Execute SlashNext's quota/status to get the quota status information
+    @:return: None
+    """
+    # 1. There is no parameter input required from Demisto
+    # 2. Get the quota status info from SlashNext API
+    response = api_quota()
+    if response.get('errorNo') != 0:
+        return
```

Comment:
what will happen if errorNo != 0?

Reply:
currently it will just do nothing, and print nothing. Is it means there is an error?

Reply:
If you the earlier function call, i.e. `api_quota()`, there a formatted error is returned to Demisto using `return_error` function call on this condition.

Reply:
@anara123 Can you have a look at the CircleCI pipeline, it seems that some other integration is failing i.e. GMail. Thanks.

Reply:
ignore that.